src/renderer/notification-ssf-hendler.ts

```diff
@@ -1,10 +1,11 @@
-import { remote } from 'electron';
```

Comment:
Maybe it's time to rename this file ```notification-ssf-handler.ts``` instead of ```notification-ssf-hEndler.ts``` ?

Reply:
Will update 😀